src/main/java/bio/terra/service/dataset/DatasetTableDao.java

```diff
@@ -118,11 +122,22 @@ public void removeTables(UUID parentId, List<String> tableNames) {
     }
   }
 
-  private void createColumns(UUID tableId, Collection<Column> columns) {
+  private void createColumnsNewTable(UUID tableId, Collection<Column> columns) {
+    createColumns(tableId, columns, 0);
+  }
+
+  public void createColumnsExistingTable(UUID tableId, Collection<Column> columns) {
+    MapSqlParameterSource params = new MapSqlParameterSource();
+    params.addValue("table_id", tableId);
+    Integer maxOrdinal = jdbcTemplate.queryForObject(sqlGetMaxColumnOrdinal, params, Integer.class);
```

Comment:
I think this will create duplicate ordinals...you probably need to do +1

Reply:
Hmm...my understanding was that this returns uses the current value of ordinal and _then_ increments by 1.  It'd be worth testing but I still think this will create a duplicate